### PR TITLE
chore (refs T33661): add missing Hooks/Selectors.

### PR DESCRIPTION
**Pull request by ahmad-demos**
https://yaits.demos-deutschland.de/T33661

- add missing Hooks/Selectors.

### CODE REVIEW

File: client/js/components/button/BackToTopButton.vue [hunk 1]
```diff
@@ -7,6 +7,7 @@
     rounded
     :style="buttonPosition"
     :text="Translator.trans('back.to.top')"
+    data-cy="DpBackToTop"
```
Comment (spiess-demos) [inline client/js/components/button/BackToTopButton.vue:10]: I'd suggest to remove the `Dp` prefix here. Also, in other places we use lowerCamelCase.

```suggestion
    data-cy="backToTop"
```

File: client/js/components/statement/assessmentTable/AssessmentTableFilter.vue [hunk 2]
```diff
@@ -238,6 +238,7 @@
             data-actionmenu>
             <button
               class="c-actionmenu__trigger"
+              data-cy="AssessmentTableFilter:Sorting"
```
Comment (spiess-demos) [inline client/js/components/statement/assessmentTable/AssessmentTableFilter.vue:241]: Please use lowerCamel for consistency (also the second part after the `:`).

File: client/js/components/statement/assessmentTable/AssessmentTableFilter.vue [hunk 3]
```diff
@@ -253,6 +254,7 @@
                 :key="`${option.value}:${i}`"
                 class="c-actionmenu__menuitem"
                 :class="{'is-active': option.value === sort.value}"
+                :data-cy="`'AssessmentTableFilter:'${option.label}`"
```
Comment (spiess-demos) [inline client/js/components/statement/assessmentTable/AssessmentTableFilter.vue:257]:
```suggestion
                :data-cy="`assessmentTableFilter:${option.label}`"
```

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Review by salisdemos — APPROVED** [review]

**Pull request merged**